internal/pubsub/pubsub.go

```diff
@@ -134,7 +143,8 @@ type Subscription struct {
 	mu sync.Mutex
 
 	// q is the local queue of messages downloaded from the server.
-	q []*Message
+	q   []*Message
```

Comment:
gofmt needed?

Reply:
The file is correctly formatted. The type lines up with the type on the line below.